website/docs/wallet/withdraw-validator.md

```diff
@@ -350,9 +350,13 @@ Once the validator is both exited as well as having its withdrawal credentials c
 
 A: After the Capella/Shanghai hardfork withdrawals will be enabled. This is expected to go live this spring, 2023.
 
+**Q: My keys were compromised, can I still withdraw?**
+
+A: You are still able to send the message as long as you have access the to mnemonic and can produce the signed `blstoexecutionchange` message to submit. Depending on where the keys were compromised there may be different protection programs to apply for to "frontrun" the compromiser. Please seek out the ethstaker community on [reddit](https://www.reddit.com/r/ethstaker/) or [discord](https://discord.gg/urhv3xby) for more details if this applies to you.
+
 **Q: I forgot my mnemonic, what can I do?**
 
-A: Unfortunately there is not much that can be done, In some niche cases where users have both their validator keystore and withdrawal private keys they can still fully withdrawal safely without the mnemonic, but unless both are in possession one would not be able to produce the signed `blstoexecutionchange` message.
+A: The mnemonic is a requirement to enabling withdraws in most cases, In some niche cases where users have both their validator keystore and withdrawal private keys they can still fully withdrawal safely without the mnemonic, but unless both are in possession one would not be able to produce the signed `blstoexecutionchange` message. It's important to stay calm and collected and continue searching or see help as needed. The ethstaker community provides an active support network on [reddit](https://www.reddit.com/r/ethstaker/) and [discord](https://discord.gg/urhv3xby)
```

Comment:
This first line sounds a bit weird, maybe: "In most cases the mnemonic is a requirement to enabling withdraws."